docs/faq.md

```
@@ -263,6 +263,69 @@ async with boxlite.SimpleBox(image="alpine:latest") as box:
+
+```bash
+# Find the most recent box
+ls -t ~/.boxlite/boxes/
+
+# Then grep its shim log
+grep -i allowNet ~/.boxlite/boxes/<id>/logs/boxlite-shim.log.<date>
+```
+
+A successful create looks like this — `allow_zones == hostnames`:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)